lib/recipe.rb

```diff
@@ -22,6 +22,9 @@
 
 windows = node['platform_family'] == 'windows'
```

Comment:
we're using `windows` and `platform?('windows')`, the latter of which I wasn't aware. Seems we should just stick with one and use it everywhere